web/components/DCApiView.vue

```diff
@@ -0,0 +1,337 @@
+        isLoading.value = true;
+        errorMessage.value = "";
+        successMessage.value = "";
+
+        console.log("Starting DC API flow...");
+        console.log("Exchange data:", props.exchangeData);
+        console.log("Workflow ID:", props.exchangeData.workflowId);
+        console.log("Exchange ID:", props.exchangeData.id);
+
+        // Get the authorization request from the server
+        const requestUrl =
+            `/workflows/${props.exchangeData.workflowId}` +
+            `/exchanges/${props.exchangeData.id}` +
+            `/dc-api/request`;
+
+        console.log("Requesting from:", requestUrl);
+
+        const { data: authRequest } = await httpClient.get(requestUrl, {
+            headers: {
+                Authorization: `Bearer ${props.exchangeData.accessToken}`,
+            },
+        });
+
+        console.log("DC API authorization request received:", authRequest);
+
+        // Use the Digital Credentials API to get credentials
+        const controller = new AbortController();
+
+        console.log("Calling navigator.credentials.get()...");
+        const credentialResponse = await navigator.credentials.get({
+            signal: controller.signal,
+            mediation: "required",
+            digital: authRequest,
+        });
+
+        console.log("DC API credential response:", credentialResponse);
+
+        if (credentialResponse) {
+            // Send the response back to the server
+            const responseUrl =
+                `/workflows/${props.exchangeData.workflowId}` +
+                `/exchanges/${props.exchangeData.id}` +
+                `/dc-api/response`;
+
+            console.log("Sending response to:", responseUrl);
+
+            const { data: result } = await httpClient.post(responseUrl, {
+                json: credentialResponse,
+                headers: {
+                    Authorization: `Bearer ${props.exchangeData.accessToken}`,
+                },
+            });
+
+            console.log("DC API response result:", result);
+            successMessage.value = "Credential presentation successful!";
+
+            // Update the exchange if needed
+            if (result.exchange) {
+                emit("replaceExchange", result.exchange);
+            }
+
+            // Store redirect data for manual continuation
+            const oidcState =
+                props.exchangeData.oidc?.state || result.exchange?.oidc?.state;
+            const oidcCode = result.exchange?.oidc?.code;
+
+            if (oidcState && oidcCode && props.rp.redirectUri) {
+                const queryParams = new URLSearchParams({
+                    state: oidcState,
+                    code: oidcCode,
+                });
+                redirectData.value = `${props.rp.redirectUri}?${queryParams.toString()}`;
+                console.log("Redirect URL prepared:", redirectData.value);
+            } else {
+                console.warn("Missing OIDC parameters for redirect:", {
```

Comment:
All oidc stuff should be handled in the login view component. Emit an event if needed.